src/tusb_option.h

```diff
@@ -489,6 +489,93 @@
     { 0x9986, 0x7523 }  /* overtaken from Linux Kernel driver /drivers/usb/serial/ch341.c */
 #endif
 
+#ifndef CFG_TUH_CDC_PL2303
+  // PL2303 is not part of CDC class, only to re-use CDC driver API
+  #define CFG_TUH_CDC_PL2303 0
+#endif
+
+#ifndef CFG_TUH_CDC_PL2303_VID_PID_QUIRKS_LIST
+  // List of product IDs that can use the PL2303 CDC driver
+  #include "class/cdc/serial/pl2303.h"
+  #define CFG_TUH_CDC_PL2303_VID_PID_QUIRKS_LIST \
+  { PL2303_VENDOR_ID, PL2303_PRODUCT_ID, PL2303_QUIRK_ENDPOINT_HACK }, \
```

Comment:
whatt is `PL2303_QUIRK_ENDPOINT_HACK` hack for ?

Reply:
oh, that's a good question...
I initially adopted it from Linux.
There seems to have been derivatives in the early days of the PL2303, where the EPs of one CDC port were distributed over two interfaces. This is no longer relevant for current derivatives. I removed these quirks.
And in pl2303_open() it is checked anyway whether the correct number of EPs is available.

I also removed PL2303_QUIRK_UART_STATE_IDX0 because it only affects the interrupt EP, which we don't use.

We should only take what is currently being sold and what we can test...